src/Chapter1.hs

```diff
@@ -640,12 +654,14 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n = if div n 10 == 0
+  then n
+  else firstDigit (div n 10)
```

Comment:
I see, that for a negative number like -19, this implementation will return the negative number instead of the last digit because of the way `div` works on negatives.

Reply:
Yes, indeed. Guess I need to get an absolute value of n first.